### PR TITLE
push images to ECR in us-west-2 also

### PR DESCRIPTION
**Pull request by ghirsch1**
https://clever.atlassian.net/browse/INFRA-3543

Tested that it works by running it on a test branch for teacher-pd-service, it made images in both regions for this commit:
https://github.com/Clever/teacher-pd-service/commit/788aba37875e4a0803bd2ddf857830ad7091ac09
https://us-west-2.console.aws.amazon.com/ecr/repositories/teacher-pd-service/?region=us-west-2
https://us-west-1.console.aws.amazon.com/ecr/repositories/teacher-pd-service/?region=us-west-1

### CODE REVIEW

File: circleci/docker-publish [hunk 1]
```diff
@@ -59,12 +59,14 @@ check_ecr_vars() {
 }
 
 ecr_login(){
+  REGION=$1
```
Comment (rgarcia) [inline circleci/docker-publish]: looks like this file prefers tabs to spaces :) 
```suggestion
	REGION=$1
```

File: circleci/docker-publish [hunk 2]
```diff
@@ -98,7 +100,8 @@ fi
 docker login -u $DOCKER_USER -p $DOCKER_PASS --email="$DOCKER_EMAIL" || docker login -u $DOCKER_USER -p $DOCKER_PASS
 if [ "$ECR_PUSH_ID" != "" ]; then
   echo "ECR access_id available. Logging into ECR..."
-  ecr_login
+  ecr_login $ECR_REGION
+  ecr_login us-west-2
```
Comment (rgarcia) [inline circleci/docker-publish]: Maybe it'd be clearer to hardcode at the top of this script two variables like `ECR_REGION_{PRIMARY,SECONDARY}` and make a clean split from the environment config in circle

File: circleci/docker-publish (hunk 1 above)
Comment (rgarcia) [reply circleci/docker-publish]: edit: maybe just this function 🤔

Comment (ghirsch1) [reply circleci/docker-publish]: weird lol, I'll just change the whole function

File: circleci/docker-publish (hunk 2 above)
Comment (ghirsch1) [reply circleci/docker-publish]: Fair point, I'll do that.

**Pull request merged**